propagators/opentelemetry-propagator-ot-trace/src/version.ts

```diff
@@ -15,4 +15,4 @@
  */
 
 // this is autogenerated file, see scripts/version-update.js
-export const VERSION = '0.13.0';
+export const VERSION = '0.13.1';
```

Comment:
Shouldn't we change the version.ts files of the others also?

Reply:
yes i was halfway through making this when a meeting started

Reply:
This afternoon I will bring over the tooling changes from the main repo which make this more automatic.